packages/nx-playwright/src/executors/playwright-executor/executor.spec.ts

```diff
@@ -1,6 +1,7 @@
 import * as utilModule from 'util';
 import executor from './executor';
 import * as startDevServerModule from './lib/start-dev-server';
+import { PlaywrightExecutorSchema } from './schema';
```

Comment:
needed type so `firefox` can be constrained to its union type (otherwise was considered as a string).